packages/attestation-service/src/requestHandlers/attestation.ts

```diff
@@ -143,11 +143,16 @@ class AttestationRequestHandler {
   }
 
   async validateAttestationRequest() {
-    const { phoneNumber, account, issuer } = this.attestationRequest
+    const { account, issuer } = this.attestationRequest
 
-    const attestationRecord = await existingAttestationRequestRecord(phoneNumber, account, issuer, {
-      logging: this.sequelizeLogger,
-    })
+    const attestationRecord = await existingAttestationRequestRecord(
+      this.identifier,
```

Comment:
Hmm, so before we were not checking for the right identifier in the db? Nice find, surprised we didn't catch this earlier. We'll need to coordinate an attestation service release to get the fix out.

Reply:
So was this causing validators to send out repeat attestations?

Reply:
Great catch @i1skn. If I remember correctly though Validators have reported seeing the "Already sent error", but judging from the bug, i'm not entirely sure how. Were  you able to repro this?

Reply:
@nambrot yeah, this error would appear in the logs. But service was returning General Error 500 instead of 422 Already sent, hence it was not possible to figure on the client side what happened. Essentially it was skipping the validation and only trigger error on the later stage, when we are trying to send an SMS. I was able to reproduce the bug and also test the fix.

Reply:
I believe Validators are showing this error message though, how would we explain that?

Reply:
@nambrot I'm also confused because I definitely remember testing and using the 'Enter codes anyway' feature we added to verification that relied on 422 responses from the Att service. Maybe what we remember working was *after* Amy removed phone numbers from these DB checks but *before* I added PGPNP support to Att service? That doesn't totally line up with my memory but I can't explain it otherwise.

Reply:
maybe the ones seeing this `Already sent` error are not running the version after I removed phone numbers from the DB